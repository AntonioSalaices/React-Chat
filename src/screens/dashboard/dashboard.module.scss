.pokemon {
  &__container {
  }

  &__header {
    border: 4px solid blue;
    border-radius: 8px;
    padding-bottom: 4em;
    padding-top: 4em;
    text-align: center;
  }

  &__headerTitle {
    text-transform: uppercase;
    font-size: 5em;
    background-image: linear-gradient(-225deg, #05236d 0%, #273aa4 29%, #e98d0e 67%, #fff800 100%);
    background-size: auto auto;
    background-clip: border-box;
    -webkit-background-clip: text;
    -webkit-text-fill-color: transparent;
    animation: textclip 2s infinite;
    letter-spacing: 10px;
  }

  &__labelImage {
    text-align: center;
  }
}

@keyframes textclip {
  to {
    background-position: 200% center;
  }
}
