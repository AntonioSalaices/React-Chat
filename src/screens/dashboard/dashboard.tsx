export function Dashboard() {
  return <div>Dashboard</div>;
}

Dashboard.displayName = 'Dashboard';
