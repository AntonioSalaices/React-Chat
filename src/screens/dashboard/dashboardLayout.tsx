import { Layout } from '@Components/Core';

export function DashboardLayout() {
  return <Layout />;
}

DashboardLayout.displayName = 'DashboardLayout';
