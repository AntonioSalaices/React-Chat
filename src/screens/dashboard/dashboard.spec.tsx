import { RenderResult, render } from '@testing-library/react';
import { Dashboard } from './dashboard';

describe('<Dashboard />', () => {
  let component: RenderResult;

  beforeEach(() => {
    component = render(<Dashboard />);
  });
  it('Should render Dashboard', () => {
    const { container } = component;

    expect(container).toMatchSnapshot();
  });
});
