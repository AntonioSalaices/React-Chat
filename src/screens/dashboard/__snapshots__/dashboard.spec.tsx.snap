// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> Should render Dashboard 1`] = `
<div>
  <div>
    Dashboard
  </div>
</div>
`;
